stdlib/mlang/compile.mc

```diff
@@ -122,8 +139,11 @@ let isSynDecl = use MLangAst in
 let isSemDecl = use MLangAst in 
   lam d. match d with DeclSem _ then true else false
 
-lang MLangCompiler = MLangAst + MExprAst
+lang DeclCompiler = DeclAst + Ast
   sem compileDecl : CompilationContext -> Decl -> CompilationResult
+end
+
+lang LetDeclCompiler = DeclCompiler + LetDeclAst + LetAst
   sem compileDecl ctx = 
   | DeclLet d -> _ok (
```

Comment:
These should not be using the `_` versions of these names, as they're intended to be private. There are a number of other similar uses as well, not only of the `_ok` function.